service/src/main/java/bio/terra/workspace/service/workspace/model/GcpCloudContext.java

```diff
@@ -19,4 +21,36 @@ public String getGcpProjectId() {
   public ApiGcpContext toApi() {
     return new ApiGcpContext().projectId(getGcpProjectId());
   }
+
+  public static final long GCP_CLOUD_CONTEXT_DB_VERSION = 1;
```

Comment:
nit: define this constant  in `GcpCloudContextV1`. This class doesn't need to keep up with a mapping of versions to  classes.